project/Dependencies.scala

```diff
@@ -23,8 +23,11 @@ object Dependencies {
     val spark3 = "3.3.2"
 
     val scala211 = "2.11.12"
-    val scala212 = "2.12.12"
-    val supportedScalaVersions: Seq[String] = Seq(scala211, scala212)
+    val scala212 = "2.12.18"
+    val scala213 = "2.13.11"
+
+    val serverScalaVersion: String = scala212
```

Comment:
Nice!

Reply:
Why is this not a Seq as well as `aqentSupportedVersions`?

Reply:
Because supporting more versions in the server is not needed imho, i.e. there will be probably just 1 instance of server, managed by 'us', deployed into Docker & ECR repo & EKS, and it will be able to communicate with the agent that might  be in any version - it's just a REST API communication between them after all

Reply:
but on the other hand, agent will be used inside of our applications' scala code, which are usually in range of Scala 2.11 - 2.13 so that must be a Seq

Reply:
Also, is there a reason for sever not being 2.13?

Reply:
> Because supporting more versions in the server is not needed imho, i.e. there will be probably just 1 instance of server, managed by 'us', deployed into Docker & ECR repo & EKS, and it will be able to communicate with the agent that might be in any version - it's just a REST API communication between them after all

@lsulak  I get it, but then you have 2 different implementations. I would have it both as Seq to have it the same across. First look you see one as Seq and one not.

Reply:
Why to have something as a different, and improper, data type, just for the sake of consistency? To the best of my knowledge, server will always be in 1 version, thus any kind of iterable doesn't make sense

Reply:
I like the way how it is.
But the DB tests will also suffice to exist under one Scala version. On the other hand the Model module is multi-versioned. So maybe a rename  of the variable to more neutral ones?